pkg/registry/apis/folders/hooks.go

```diff
@@ -66,6 +68,22 @@ func (b *FolderAPIBuilder) beginUpdate(ctx context.Context, obj runtime.Object,
 	}, nil
 }
 
+func (b *FolderAPIBuilder) afterDelete(obj runtime.Object, _ *metav1.DeleteOptions) {
+	ctx := context.Background()
+	log := logging.FromContext(ctx)
```

Comment:
This code makes no sense. `FromContext` only makes sense if the context has already a logger attached to it, which is impossible here as it's a new context.

Reply:
It also returns `DefaultLogger` if the passed context contains no logger.

Nonetheless, cleaned up here: https://github.com/grafana/grafana/pull/111649/commits/c3e365a91fc7ad9609bbf8816d2ec1d5334b42bd